### ISSUE TITLE
Need help with API Tokens

### ISSUE THREAD

**Issue by ElgerElg** [ashakoen/next-js-flux1-replicate#1]
Greetings! In the hope that they will answer me here, I tell them about my problems. I'm new to AI, trying to create a personal image generation for me and friends. Following the instructions in next-js-flux1-replicate, I came across the error: 

"Error from Replicate API: {
title: 'Invalid version or not allowed',
  detail: "The specified version does not exist (or perhaps you don't have permission to use it?)",
status: 422"

 I do not know what to do. I hope for help.

**Comment by ashakoen** [comment]
Sorry about that! You need a Replicate API key to generate images. You can get one here:

https://replicate.com/account/api-tokens

Once you have a Replicate API key, you enter it into the app. See below:

![CleanShot 2024-09-17 at 16 29 13](https://github.com/user-attachments/assets/05d1225b-a6f7-4498-b411-16b8d23f396e)

**Comment by ElgerElg** [comment]
Yeah I did it but it gives me error. Maybe to Replicate need to pay or something? Maybe need setup billing? Thank you for reply

**Comment by ashakoen** [comment]
Yes, you would need to setup billing at Replicate since the application uses their infrastructure to do the FLUX.1 predictions.

**Comment by ElgerElg** [comment]
Ah thank you very much I will try and say if needs

**Comment by ElgerElg** [comment]
Hello! So, I linked my visa card to replicate but in this site I get this error again... Token is entered

"Error from Replicate API: {
title: 'Invalid version or not allowed',
detail: "The specified version does not exist (or perhaps you don't have permission to use it?)",
status: 422"

**Comment by ashakoen** [comment]
How are you running the code, locally? Also, are you using the Private LoRA feature or just trying to generate images using FLUX.1 dev or schnell base models?

**Comment by ashakoen** [comment]
Ok. Thanks for confirming. I do see an issue if you try to use the FLUX.1 dev model without a LoRA. I built the tool specifically for LoRA use, so let me see if I can get a fix put in asap!
